public/js/action-list.js

```diff
@@ -106,6 +122,20 @@
                 url = $list.attr('data-icinga-multiselect-url') + '?(' + filters.toArray().join('|') + ')';
             }
 
+            if (footer.length) {
```

Comment:
That a footer exists, doesn't mean it's a multiselect list. Please check explicitly for a multiselect list.